### ISSUE TITLE
Kubernetes Cluster Topology not being displayed

### ISSUE THREAD

**Issue by maxamillion** [cockpit-project/cockpit#2548]
When logging into the web console and selecting the "Cluster" tab, the "Topology" tab causes an "oops" to be registered in the top right corner of the UI.

Cockpit version cockpit-0.63-2.el7.x86_64

The following is the error logged to the JS console (with cockpit-debuginfo installed).
```
"Error: client.wait is not a function
@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/topology.js:54:17
invoke@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:4188:14
instantiate@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:4196:27
$ControllerProvider/this.$get</<@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:8465:18
ngViewFillContentFactory/<.link@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:27159:26
invokeLinkFn@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:8222:9
nodeLinkFn@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:7732:1
compositeLinkFn@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:7081:13
publicLinkFn@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:6960:30
createBoundTranscludeFn/boundTranscludeFn@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:7099:1
controllersBoundTransclude@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:7759:18
update@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:27117:25
$RootScopeProvider/this.$get</Scope.prototype.$broadcast@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:14723:15
commitRoute/<@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:26800:15
processQueue@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:13192:27
scheduleProcessQueue/<@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:13208:27
$RootScopeProvider/this.$get</Scope.prototype.$eval@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:14404:16
$RootScopeProvider/this.$get</Scope.prototype.$digest@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:14220:15
$RootScopeProvider/this.$get</Scope.prototype.$apply@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:14509:13
$LocationProvider/this.$get</<@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:11367:13
createEventHandler/eventHandler@https://192.168.122.33:9090/cockpit/$7f8ba3ba5fb4427be3df0d36c26fd9ab241a4f32/kubernetes/angular.js:3017:9
" "<div id="content" class="container-fluid ng-scope" ng-view="" ng-if="!curtains">"
```

**Comment by stefwalter** [comment]
I hate to do this, but that code has changed. Could you try 0.69? from here: 

https://copr.fedoraproject.org/coprs/sgallagh/cockpit-preview/
https://copr.fedoraproject.org/coprs/sgallagh/cockpit-preview/build/107303/

**Comment by stefwalter** [comment]
Verified. This is a duplicate of #2462 which was fixed by ef2257bd24ed21129bf89e7edd4632f94f303707 in 0.64 and later.
